CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.11.0 - 2018-01-19
```

Comment:
This date is ...not right

It's no longer 2018 and it's the 15th (unless you were planning on delaying for some reason??)